Course_Part_06/src/Employee.java

```diff
@@ -0,0 +1,36 @@
+public class Employee {
+
+    int id;
+    String firstName;
+    String lastName;
+    int salary;
+
+    Employee(int id, String firstName, String lastName, int salary){
+        this.id = id;
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.salary = salary;
+    }
+
+    public int getAnnualSalary() {
+        return this.salary * 12;
+    }
+
+    public void printAnnualSalary(){
+        System.out.println("Annual salary is: " + this.getAnnualSalary());
```

Comment:
Onnodig gebruik van keyword 'this' en methode aanroepen.

Je kan direct het field 'salary' gebruiken.